### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by Pleasesendhelp101** [dnschneid/crouton#4536] [Q&A]
trying to install kali linux on my acer chromebook spin 311 not sure if that matters but i get this error message at the end.
![Screenshot 2021-07-05 2 11 36 PM](https://user-images.githubusercontent.com/86986298/124518392-1bbd6b00-dd9b-11eb-89d5-68f6b0676924.png)

**Comment by ghost** [comment]
Duplicate of [#4450](https://github.com/dnschneid/crouton/issues/4450).
